docs/settings/ingest-manager-settings.asciidoc

```diff
@@ -0,0 +1,65 @@
+[role="xpack"]
+[[ingest-manager-settings-kb]]
+=== Ingest Manager settings in {kib}
+++++
+<titleabbrev>Ingest Manager settings</titleabbrev>
+++++
+
+You can configure `xpack.ingestManager` settings in your `kibana.yml`. 
+By default Ingest Manager is not enabled you need to enable it. Also to you use Fleet, you need to configure {kib} and {es} hosts.
+
+[[general-ingest-manager-settings-kb]]
+==== General Ingest Manager settings
+
+[cols="2*<"]
+|===
+| `xpack.ingestManager.enabled` {ess-icon}
+  | Set to `true` to enable Ingest Manager. 
+|===
+
+[cols="2*<"]
+|===
+| `xpack.ingestManager.epm.enabled` {ess-icon}
+  | Set to `true` (default) to enable the package manager. 
+|===
+
+[cols="2*<"]
+|===
+| `xpack.ingestManager.fleet.enabled` {ess-icon}
+  | Set to `true` (default) to enable Fleet. 
+|===
+
+[[ingest-manager-data-visualizer-settings]]
+
+==== EPM settings
```

Comment:
```suggestion
==== {package-manager} settings
```

Reply:
I am wondering if I should be using {package-registry} here instead of {package-manager}?